docs/core/docker/index.md

```diff
@@ -19,4 +19,4 @@ The following tutorials are available for learning about using Docker with .NET
 - [Building Docker Images for .NET Core Applications](building-net-docker-images.md)
 - [Visual Studio Tools for Docker](visual-studio-tools-for-docker.md)
 
-For tutorials about developing ASP.NET Core web applications, we suggest you head over to [ASP.NET Core documentation](https://docs.asp.net).
\ No newline at end of file
+For tutorials about developing ASP.NET Core web applications, we suggest you head over to [ASP.NET Core documentation](https://docs.microsoft.com/aspnet/core/).
```

Comment:
"head over" is a bit too informal, and I can't imagine that it localizes well; "we suggest you head over to" --> "see"